### PR TITLE
LRDOCS-9466 ConfigurationFormRenderer

### PR DESCRIPTION
**Pull request by mtokudome**

### CODE REVIEW

File: docs/dxp/latest/en/developing-applications/core-frameworks/configuration-framework/configuration-form-renderer/resources/liferay-b7r2.zip/b7r2-web/src/main/java/com/acme/b7r2/web/internal/configuration/admin/util/B7R2ConfigurationFormRenderer.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+								_servletContext, httpServletRequest, httpServletResponse,
+								"/b7r2.jsp");
+
+    }
+
+		@Reference
+		private ConfigurationProvider _configurationProvider;
+
+		@Reference(
+			target = "(osgi.web.symbolicname=com.acme.b7r2.web)", unbind = "-"
```
Comment (drewbrokke) [inline docs/dxp/latest/en/developing-applications/core-frameworks/configuration-framework/configuration-form-renderer/resources/liferay-b7r2.zip/b7r2-web/src/main/java/com/acme/b7r2/web/internal/configuration/admin/util/B7R2ConfigurationFormRenderer.java:53]: I don't think `unbind` is needed here.

File: docs/dxp/latest/en/developing-applications/core-frameworks/configuration-framework/configuration-form-renderer/resources/liferay-b7r2.zip/b7r2-web/src/main/java/com/acme/b7r2/web/internal/configuration/admin/util/B7R2ConfigurationFormRenderer.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			return params;
+    }
+
+    @Override
+    public void render(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse)
+            throws IOException {
+
+						jspRenderer.renderJSP(
+								_servletContext, httpServletRequest, httpServletResponse,
+								"/b7r2.jsp");
```
Comment (drewbrokke) [inline docs/dxp/latest/en/developing-applications/core-frameworks/configuration-framework/configuration-form-renderer/resources/liferay-b7r2.zip/b7r2-web/src/main/java/com/acme/b7r2/web/internal/configuration/admin/util/B7R2ConfigurationFormRenderer.java:45]: Is this code running at all? If you put a `System.out.println("Hello World");` statement here, does it print to the console?

File: docs/dxp/latest/en/developing-applications/core-frameworks/configuration-framework/configuration-form-renderer/resources/liferay-b7r2.zip/b7r2-web/src/main/java/com/acme/b7r2/web/internal/settings/definition/B7R2ConfigurationBeanDeclaration.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+@Component(service = ConfigurationBeanDeclaration.class)
+public class B7R2ConfigurationBeanDeclaration
+	implements ConfigurationBeanDeclaration {
+
+	@Override
+	public Class<?> getConfigurationBeanClass() {
+		return B7R2Configuration.class;
+	}
+
+}
```
Comment (drewbrokke) [inline docs/dxp/latest/en/developing-applications/core-frameworks/configuration-framework/configuration-form-renderer/resources/liferay-b7r2.zip/b7r2-web/src/main/java/com/acme/b7r2/web/internal/settings/definition/B7R2ConfigurationBeanDeclaration.java:18]: This class is only necessary if you are going to retrieve this configuration using `ConfigurationProvider`.

**Review by drewbrokke — COMMENTED** [review]
Hey @mtokudome, I don't see any obvious reason why it's not rendering, but I left a suggestion to try to print out a statement too see if the code is running at all.

**Comment by mtokudome** [comment]
Now published: https://learn.liferay.com/dxp/latest/en/building-applications/core-frameworks/configuration-framework/configuration-form-renderer.html

**Pull request closed without merging**